Firebase/Auth/Source/AuthProviders/Phone/FIRPhoneAuthProvider.m

```diff
@@ -403,12 +409,16 @@ - (void)reCAPTCHAURLWithCompletion:(FIRReCAPTCHAURLCallBack)completion {
     NSString *bundleID = [NSBundle mainBundle].bundleIdentifier;
     NSString *clienID = _auth.app.options.clientID;
     NSString *apiKey = _auth.app.options.APIKey;
+    _eventID = [NSString stringWithFormat:@"%c%c", 64 + arc4random_uniform(26), 64 +
+        arc4random_uniform(26)];
+
     NSMutableDictionary *urlArguments = [[NSMutableDictionary alloc] initWithDictionary: @{
       @"apiKey" : apiKey,
       @"authType" : kAuthTypeVerifyApp,
       @"ibi" : bundleID,
       @"clientId" : clienID,
-      @"v" : [FIRAuthBackend authUserAgent]
+      @"v" : [FIRAuthBackend authUserAgent],
+      @"eventId" : _eventID
```

Comment:
Please add the trailing comma after the last item so the line won't be marked as changed next time a new item is  appended after it.

Reply:
Done